source/daplink/drag-n-drop/flash_decoder.c

```diff
@@ -351,7 +361,11 @@ static bool flash_decoder_is_at_end(uint32_t addr, const uint8_t *data, uint32_t
         case FLASH_DECODER_TYPE_TARGET:
             //only if we are sure it is a bin for the target; without check unordered hex files will cause to terminate flashing
             if (flash_type_target_bin && g_board_info.target_cfg) {
+#ifdef TARGET_MCU_PSOC6
+                end_addr = get_flash_end();
+#else				
```

Comment:
This computation can be part of prerun_board_config() under board_info api

Reply:
I agree. Use of the get_flash_end() function is eliminated